include/sparrow/layout/primitive_array.hpp

```diff
@@ -55,6 +53,21 @@ namespace sparrow
         using iterator_tag = std::random_access_iterator_tag;
     };
 
+    /**
+     * Array of values having the same binary size.
```

Comment:
So there can be multiple types as long as they have the same binary size?

Reply:
No, they all have the same type.